tools/test_fail_live_repair.sh

```diff
@@ -120,67 +133,100 @@ count=1
 while [[ $count -le $loops ]]; do
     SECONDS=0
     choice=$((RANDOM % 3))
+    ds_pid=$(./target/release/dsc cmd pid -c "$choice" | sed 's/[^0-9]*//g')
 
     # Clear the log on each loop
     echo "" > "$test_log"
     echo "New loop starts now $(date) faulting: $choice" | tee -a "$test_log"
+    echo "Downstairs client $choice has pid: $ds_pid" | tee -a "$test_log"
+
     # Start sending IO.
     "$crucible_test" generic "${args[@]}" --continuous \
         -q -g "$gen" --verify-out "$verify_file" \
         --verify-in "$verify_file" \
-        --control 127.0.0.1:7777 \
+        --control 127.0.0.1:7890 \
```

Comment:
Nit: why did the control port change?

Reply:
I was getting port collisions with something else on atrium, so I just moved this to
pick a different port.

Reply:
👍🏻  ship it